test.html

```diff
@@ -3,72 +3,99 @@
 <head>
     <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
     <script>
-        function reload() {
-            $.ajax({
-                url: "/api/nk/a2_logistics",
-                type: "GET",
-                cache: false,
-                headers: { "Accept": "application/json" },
-                timeout: 3000,
-                success: function (data) {
-                    var obj = JSON.parse(data);
-
-                    // 通信インジケーター点滅処理
-                    var $comm = $("#comm");
-                    $comm.removeClass("p0");
-                    $comm.removeClass("p1");
-                    $comm.addClass("p" + (tick++ & 0x1));
-                    // 次の通信を仕掛ける
-
-                    }
-
-            });
-
-           return data = {
-                "status": "test",
-                "now": 30,
-                "next_at": 1,
-                "parts": [
-                    { "car_number": "1", "stop_location": "here", "part_number": "12100 3X00A", "part_name": "F", "maker": "日立", "box_color": "red", "required_number": 1, "deadline": "09:30", "inputs": 1 },
-                    { "car_number": "1", "stop_location": "there", "part_number": "12101 3X00A", "part_name": "D", "maker": "RYOBI", "box_color": "yellow", "required_number": 1, "deadline": "09:31", "inputs": 1 },
-                    { "car_number": "2", "stop_location": "there", "part_number": "12101 3X00A", "part_name": "D", "maker": "RYOBI", "box_color": "yellow", "required_number": 1, "deadline": "09:31", "inputs": 1 },
-                    { "car_number": "3", "stop_location": "there", "part_number": "12101 3X00A", "part_name": "D", "maker": "RYOBI", "box_color": "yellow", "required_number": 1, "deadline": "09:31", "inputs": 1 }
-
-                ]
-            }
-
-        }
-        $(document).ready(function() {
-           init();
-        })
-
-        function init() {
-            var data = reload();
-        }
-
-
-
-
-    </script>
+    function reload() {
```

Comment:
the code formatter in my text editor auto formatted the JS code. sorry bout that!